js/index.js

```diff
@@ -1,42 +1,90 @@
 // ITERATION 1
 
 function updateSubtotal(product) {
-  console.log('Calculating subtotal, yey!');
-
+  //console.log('Calculating subtotal, yey!');
+  const price = (product.querySelector('.price span').innerHTML);
```

Comment:
You don't need extra parentheses, you could just write it `const price = product.querySelector('.price span').innerHTML`